examples/js/ada/send-withdraw-rewards-transaction.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Send an unstaking transaction from a multi-sig wallet at BitGo.
+ *
+ * Copyright 2022, BitGo, Inc.  All Rights Reserved.
+ */
+const BitGoJS = require('bitgo');
```

Comment:
Let's show the example using `@bitgo/sdk-api` and `@bitgo/sdk-coin-ada`

Reply:
So instead of importing the entire bitgo module, import @bitgo/sdk-api and @bitgo/sdk-coin-ada to do the same thing?

Reply:
I guess same question as the other comment, is this an overarching change we're making to the scripts?

Reply:
Yes, we want to default to the modularized approach for users.